### ISSUE TITLE
Fix CssSyntaxError for single quotes in comment and value

### ISSUE THREAD

**Issue by dmowski** [stylelint/stylelint#3943]
When add comment 
```
// With single' quote
```

4:13
error
Unclosed string (CssSyntaxError)

> Which rule, if any, is the bug related to?
```less
// Tab's styles
a:before {
  content: '';
}
```
```json
{
  "rules": {}
}
```
Run from https://stylelint.io/demo/

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

e.g. "Yes, It's related to Less."

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

```shell
3:13
error
Unclosed string (CssSyntaxError)
```

**Comment by jeddy3** [comment]
@dmowski Thanks for the report and for using the template.

I'm unable to reproduce this when I select "less" from the syntax options in the demo:

<img width="410" alt="screenshot 2019-02-06 at 10 05 16" src="https://user-images.githubusercontent.com/808227/52334217-12c3e600-29f7-11e9-8a15-ab8e834a129d.png">

I'll close for now, but feel free to reopen if selecting the syntax doesn't resolve the issue for you.

**Comment by dmowski** [comment]
Right now, dont reproduces on demo page

But still reprodused on local PC.

Repository with example
https://github.com/dmowski/stylelint-bug-3943

```
"stylelint": "9.10.1"
```

```
// Dobule"quote
a:before {
  content: ""; //required str with Dobule quote for reproduce bug
}

// single'quote
a:before {
  content: ''; //required str with single quote for reproduce bug
}
```

**Comment by hudochenkov** [comment]
Looks like it's an issue in [postcss-less](https://github.com/shellscape/postcss-less), which we use for parsing Less. Would you mind reporting there, please?

**Comment by dmowski** [comment]
https://github.com/shellscape/postcss-less/issues/134

**Comment by jeddy3** [comment]
This [appears to be an issue with stylelint itself](https://github.com/shellscape/postcss-less/issues/134#issuecomment-462152221) rather than the Less parser.

This issue is labeled as "help wanted" if anyone has time to investigate what's going on.

**Comment by PabloFNK** [comment]
As extra input for anyone working on this. This was happening to us in Windows machines but not in MAC ones.

**Comment by jwilsson** [comment]
This might actually be caused by https://github.com/shellscape/postcss-less/issues/136, but we've got  a fix pending so this should be resolved shortly.

**Comment by jwilsson** [comment]
This is fixed upstream in `postcss-less@3.1.3` so a fresh `npm install` should take care of it automatically.
